profiles/package.mask

```diff
@@ -164,7 +164,6 @@ net-irc/jircii
 net-irc/miau
 net-mail/hotwayd
 net-mail/popick
-net-mail/poppassd_ceti
```

Comment:
Please make this change in a separate commit, this file is everliving and will probably change a thousand times before this gets merged. Having it in a separate commit makes my life so much easier :)

Reply:
Done.